src/azure-cli/azure/cli/command_modules/backup/custom.py

```diff
@@ -23,14 +23,68 @@
     BackupResourceVaultConfig, BackupResourceVaultConfigResource, DiskExclusionProperties, ExtendedProperties
 
 from azure.cli.core.util import CLIError
+from azure.cli.core.azclierror import RequiredArgumentMissingError, InvalidArgumentValueError
 from azure.cli.command_modules.backup._client_factory import (
     vaults_cf, backup_protected_items_cf, protection_policies_cf, virtual_machines_cf, recovery_points_cf,
     protection_containers_cf, backup_protectable_items_cf, resources_cf, backup_operation_statuses_cf,
     job_details_cf, protection_container_refresh_operation_results_cf, backup_protection_containers_cf,
-    protected_items_cf, backup_resource_vault_config_cf)
+    protected_items_cf, backup_resource_vault_config_cf, recovery_points_crr_cf, aad_properties_cf,
+    cross_region_restore_cf, backup_crr_job_details_cf, crr_operation_status_cf, backup_crr_jobs_cf)
 
 logger = get_logger(__name__)
 
+# Mapping of workload type
+secondary_region_map = {"eastasia": "southeastasia",
+                        "southeastasia": "eastasia",
+                        "australiaeast": "australiasoutheast",
+                        "australiasoutheast": "australiaeast",
+                        "australiacentral": "australiacentral2",
```

Comment:
AKS also has a similar [map](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/custom.py#L3004). It is the same or may be different for different RPs?

Reply:
No, it seems to be a different map.